services/server/test/unit/VerificationService.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { VerificationService } from "../../src/server/services/VerificationService";
+import nock from "nock";
+import fs from "fs";
+import path from "path";
+import { expect } from "chai";
+import { findSolcPlatform } from "@ethereum-sourcify/lib-sourcify/test/compiler/solidityCompiler";
+import config from "config";
+import rimraf from "rimraf";
+
+describe("VerificationService", function () {
+  beforeEach(function () {
+    // Clear any previously nocked interceptors
+    nock.cleanAll();
+  });
+
+  afterEach(function () {
+    // Ensure that all nock interceptors have been used
+    nock.isDone();
+  });
+
+  it("should initialize compilers", async function () {
+    before(function () {
+      rimraf.sync(config.get("solcRepo"));
+      rimraf.sync(config.get("solJsonRepo"));
+    });
```

Comment:
The `before` hook should not be called inside a `it` block. The functions should rather be called directly here without wrapping in the before hook.